avm/src/main.rs

```diff
@@ -3,20 +3,21 @@ use std::env;
 use std::fs::read;
 use std::path::Path;
 
-use anycloud::common::get_agz_file_b64;
-use anycloud::deploy;
-use anycloud::oauth::authenticate;
 use base64;
 use clap::{crate_name, crate_version, App, AppSettings, SubCommand};
 use tokio::runtime::Builder;
 
 use crate::compile::compile::compile;
 use crate::daemon::daemon::{start, CLUSTER_SECRET};
+use crate::deploy::common::get_agz_file_b64;
+use crate::deploy::deploy as inner_deploy;
```

Comment:
`deploy_mod`?

Reply:
Discussed offline, going for `use crate::cloud::deploy` instead.